### ISSUE TITLE
Remove/adjust duplicate loadout transfer logic in headless

### ISSUE THREAD

**Issue by freghar** [acemod/ACEX#256]
**Is your enhancement related to a problem?**
CBA implemented a "naked unit bug workaround" in https://github.com/CBATeam/CBA_A3/pull/1406, making the ACEX `acex_headless_transferLoadout` setting obsolete (the way I understood it per the long discussion in the PR).

While the functionality is not exactly the same, it is similar enough to where it might not be worth the maintenance cost, so I would suggest removing it.

**Solution you'd like:**
I don't particularly care either way.

**Alternatives you've considered:**
Keeping both settings separate and maybe throwing a warning if both are enabled (assuming this could cause issues).

**Comment by jonpas** [comment]
Yes, this should be done. I was focusing on https://github.com/acemod/ACE3/pull/8415 in the meantime though.
